agents/ocf/HealthCPU.in

```diff
@@ -121,6 +120,10 @@ healthcpu_monitor() {
     if [ -f "${OCF_RESKEY_state}" ]; then
 
         IDLE=$(top -b -n2 | grep Cpu | tail -1 | awk -F",|.[0-9][ %]id" '{ print $4 }')
+        if [ $? -ne 0 ]; then
+            ocf_log err "Could not get CPU idleness"
+            return $OCF_DEGRADED
```

Comment:
> Degraded success. A "monitor" action may return this if the service is found to be properly active, but in such a condition that future failures are more likely.

Questionable here, but this agent's purpose is a little different from that of most agents. I guess if we can't get the idleness, we might be more likely to fail in the future.

Alternatively we could declare a failure after some number of consecutive failures to get idleness. I don't know that I like that either.

Reply:
Thanks, I actually did see that. That doesn't mean it aligns with the OCF spec. But there's a reasonable argument in favor of it and it doesn't clearly go against the spec. It's probably fine.